### PR TITLE
[#1330] Update device registry, release notes and documentation.

### PR DESCRIPTION
**Pull request by kaniyan**
This PR provides updated release notes and documentation for issue #1330.

### CODE REVIEW

File: site/documentation/content/api/Tenant-API.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ The table below provides an overview of the standard members defined for the JSO
...
 | *defaults*               | *no*      | *object*      |  Arbitrary *default* properties for devices belonging to the tenant. The properties can be used by protocol adapters to augment downstream messages with missing information, e.g. setting a default content type or time-to-live. |
 | *enabled*                | *yes*     | *boolean*     | If set to `false` the tenant is currently disabled. Protocol adapters MUST NOT allow devices of a disabled tenant to connect and MUST NOT accept data published by such devices. |
+| *minimum-message-size*   | *no*      | *number*      | The minimum message size in bytes. If it is set then the payload size of the telemetry, event and command messages are calculated in accordance with the configured value and then reported to the metrics. See [Metrics]({{< relref "Tenant-API.md/#minimum-message-size" >}}) for more details.|
```
Comment (sophokles73) [inline site/documentation/content/api/Tenant-API.md]: this also needs to be added to the Device Management API. We also need to make sure that this property is supported by the example Device Registry implementation.

Comment (kaniyan) [reply site/documentation/content/api/Tenant-API.md]: @sophokles I have pushed a new commit to update the _Device Management API_ and _Device Registry_ implementation.

File: service-base/src/main/java/org/eclipse/hono/service/management/tenant/Tenant.java [hunk 2]
```diff
@@ -41,6 +41,10 @@
     @JsonInclude(Include.NON_EMPTY)
     private List<Adapter> adapters = new LinkedList<>();
 
+    @JsonProperty("minimum-message-size")
```
Comment (sophokles73) [inline service-base/src/main/java/org/eclipse/hono/service/management/tenant/Tenant.java]: IMHO this should be a constant declared in `RegistryManagementConstants` ...

File: service-base/src/main/java/org/eclipse/hono/service/management/tenant/Tenant.java [hunk 3]
```diff
@@ -79,6 +83,14 @@ public void setAdapters(final List<Adapter> adapters) {
         this.adapters = adapters;
     }
 
+    public Integer getMinimumMessageSize() {
+        return minimumMessageSize;
+    }
+
+    public void setMinimumMessageSize(final Integer minimumMessageSize) {
+        this.minimumMessageSize = minimumMessageSize;
```
Comment (sophokles73) [inline service-base/src/main/java/org/eclipse/hono/service/management/tenant/Tenant.java]: prevent negative values?

File: service-base/src/main/java/org/eclipse/hono/service/management/tenant/Tenant.java (hunk 2 above)
Comment (kaniyan) [reply service-base/src/main/java/org/eclipse/hono/service/management/tenant/Tenant.java]: Done. Pls take a look.

File: service-base/src/main/java/org/eclipse/hono/service/management/tenant/Tenant.java (hunk 3 above)
Comment (kaniyan) [reply service-base/src/main/java/org/eclipse/hono/service/management/tenant/Tenant.java]: Done

**Pull request merged**